tests/manage/pv_services/test_ceph_capacity_recovery.py

```diff
@@ -0,0 +1,269 @@
+import logging
+
+from ocs_ci.ocs.perftests import PASTest
+
+from ocs_ci.ocs import node, constants
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.exceptions import (
+    PodNotCreated,
+    CephHealthException,
+    TimeoutExpiredError,
+)
+from ocs_ci.ocs.resources import pvc, ocs
+from ocs_ci.ocs.cluster import (
+    get_percent_used_capacity,
+    get_osd_utilization,
+)
+from ocs_ci.utility.utils import TimeoutSampler, ceph_health_check_base
+
+from ocs_ci.ocs.benchmark_operator_fio import get_file_size, BenchmarkOperatorFIO
+
+log = logging.getLogger(__name__)
+
+
+def check_health_status():
+    """
+    Exec `ceph health` cmd on tools pod to determine health of cluster and logs the results
+
+    Returns:
+        boolean: True if HEALTH_OK
+
+    """
+    try:
+        status = ceph_health_check_base()
+        if status:
+            log.info("Health check passed")
+        else:
+            log.info("Health check failed")
+        return status
+    except CephHealthException as e:
+        # skip because ceph is not in good health
+        log.info(f"Ceph health exception received: {e}")
+        return False
```

Comment:
you can use the function : [ceph_health_check()](https://github.com/red-hat-storage/ocs-ci/blob/0e2db8e7846b08e1d621176fd1170b2e4b9fa3a2/ocs_ci/utility/utils.py#L2033) - this is code duplication

Reply:
Sure, eventually I call to ceph_health_check_base library function, but I didn't want to call ceph_health_check since I wanted to enhance it with logging (and run the base only once , without tries and delays)

Reply:
can we enhance the https://github.com/red-hat-storage/ocs-ci/blob/0e2db8e7846b08e1d621176fd1170b2e4b9fa3a2/ocs_ci/utility/utils.py#L2033 function itself to avoid duplication?